lib/sass.dart

```diff
@@ -326,7 +326,8 @@ Future<CompileResult> compileStringToResultAsync(String source,
 ///
 /// {@category Compile}
 @Deprecated("Use compileToResult() instead.")
-String compile(String path,
+String compile(
```

Comment:
I'm not sure if these changes should stay. Please let me know if I should reset this file :)